addons/udes_load_testing/tests/README.md

```diff
@@ -0,0 +1,48 @@
+# UDES load testing
+
+Provides load tests for UDES
+
+## Running tests
+
+Load tests can be run using the normal odoo unit tests framework:
+To run all tests
+
+``` bash
+odoo-bin -i udes_stock,udes_load_testing --test-enable
```

Comment:
Doesn't run via odoo-bin, only via udes, according to some manual testing I just did.

Reply:
This can be left as-is, as per discussion.